docs/contributing/how-to-pitch-gatsby.md

```diff
@@ -0,0 +1,24 @@
+---
+title: How to Pitch Gatsby
+---
+
+If you love Gatsby.js as much as we do, you might want to pitch your colleagues, managers or anyone in your life about its value. As such, we've put together some talking points that could help you make a case for building with Gatsby:
+
+Build blazing-fast websites without the hosting hassles of managing complex, costly infrastructure or fear that your site might go down. **Gatsby is a website development tool for React, the most modern way to build reliable, lightning fast websites.**
+
+## A website built with Gatsby is:
+
+- **Fast:** Think rocket ship vs. Winnebago. Gatsby takes fast to a whole new level with
+  websites that live on the edge—right where your customers are. Pages load in milliseconds
+  rather than seconds. Gatsby lets teams create lightning-fast, content-driven websites without
+  needing to become performance experts.
+
+- **Simple:** No servers. No databases. Traditional hosting is time consuming, costly, complex, and puts your site at risk of crashing if your content goes viral. Avoid the pitfalls of hosting with a serverless architecture. A site that never goes down because it’s built on the edge, instantly scaling for traffic spikes.
+
+- **Trusted:** Leverage the power of the latest web technologies including React, Webpack, GraphQL, modern JavaScript, and more. Get the best of both worlds—all the power and conveniences from React and its ecosystem with a really slick and responsive user experience.
+
+- **Flexible with BYOC (Bring your own content):** Build websites quickly and deploy them safely, no matter where your content lives. Load data from your CMS, static files, a database, or from multiple places at once. Your marketing team is happy because they can use the content management tools they’re familiar with (WordPress, Drupal, Contentful, Shopify).
```

Comment:
We've moved to "content mesh" rather than "BYOC" framing

Reply:
Good tip! I've updated it.